types/dompurify/index.d.ts

```diff
@@ -83,13 +83,13 @@ declare namespace DOMPurify {
 
     interface SanitizeElementHookEvent {
         tagName: string;
-        allowedTags: string[];
+        allowedTags: { [key: string]: boolean };
```

Comment:
as you checked tests option, can we amend the test file to cover that change? Thanks!
```ts
//test the 'SanitizeElementHookEvent' type is publicly accessible.
dompurify.addHook('uponSanitizeElement', (currentNode: Element, event: DOMPurify.SanitizeElementHookEvent) => {
    if (currentNode.nodeName && currentNode.nodeName.match(/^\w+-\w+$/) && !event.allowedTags[event.tagName]) {
        event.allowedTags[event.tagName] = true;
    }
});
```

Reply:
Done.